Lib/TSCompiler/ProjectOptions.cs

```diff
@@ -465,7 +466,7 @@ public void InitializeTranslationDb(string specificPath = null)
             TranslationDb.AddLanguage(DefaultLanguage ?? "en-us");
             if (specificPath == null)
             {
-                TranslationDb.LoadLangDbs(PathUtils.Join(Owner.Owner.FullPath, "translations"));
+                TranslationDb.LoadLangDbs(PathToTranslations ?? PathUtils.Join(Owner.Owner.FullPath, "translations"));
```

Comment:
Why not easier:
`TranslationDb.LoadLangDbs(PathUtils.Join(Owner.Owner.FullPath, PathToTranslations ?? "translations"));`

And than you can revert modification of Join and Normalize methods.